maestro-cli/src/main/java/maestro/cli/api/ApiClient.kt

```diff
@@ -86,6 +89,17 @@ class ApiClient(
         }
     }
 
+    private fun getAgent(): String {
+        for (ciVar in knownCIEnvVars) {
+            try {
+                val value = System.getenv(ciVar)
+                if (value.isNotEmpty()) return "ci"
```

Comment:
What if we have `CI=false`?

Reply:
Good call, didn't think about that case. Do you think it makes more sense to consider it being ci iff `CI=true` or should we consider it being ci if `CI != false`?

Reply:
That and also if set to "0"